packages/body/README.md

```diff
@@ -111,6 +111,17 @@ This is also known as a Prose component.
 				var i: Integer; begin i := <mark>1.1</mark>; end.
 			</code>
 		</pre>
+		<figure>
```

Comment:
Blockquotes have a border width applied, so I added this so we can see this in action.

Quote taken from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote